inspirehep/records/cli.py

```diff
@@ -178,6 +178,8 @@ def _finished_tasks_count():
         fg=color,
     )
     if failures:
-        logger.error(f"Got {len(failures)} during the recalculation process:")
+        LOGGER.error(
+            "Got '%d' failures during the recalculation process:", len(failures)
+        )
```

Comment:
Here we send to Sentry an error that says we have had errors, which is not very helpful plus every error we actually got, which instead it is helpful. I'd remove this log, it is repetitive and doesn't add anything

Reply:
or maybe a warning?

Reply:
warning it is :+1: